collect_app/src/androidTest/java/org/odk/collect/android/feature/instancemanagement/AutoSendTest.java

```diff
@@ -65,4 +65,25 @@ public void whenFormHasAutoSend_fillingAndFinalizingForm_sendsFormAndNotifiesUse
         notificationDrawerRule.open()
                 .assertAndDismissNotification("ODK Collect", "ODK auto-send results", "Success");
     }
+
+    @Test
+    public void whenFormIsFinalized_enablingAutoSend_schedulesSubmitFinalizedForm() {
```

Comment:
I think we have doubled up test coverage here between this and the test in `FormManagementPreferencesFragmentTest`. More than happy to discuss different approaches (here or on Slack), as obviously there are no right answers here, but I'd suggest removing this test and sticking with the coverage at the Fragment level for this feature.

Reply:
As a side note, I would often start with a test like this for a feature and then if I discover I can write a more focused test that covers all the same bases at a lower level later, I'd get rid of my high level one. I've found I've been doing that more in Collect than I would have in other apps (where I might just have kept both tests) as the app has so many features that the sheer number of tests is hard to keep under control.

Reply:
This makes sense, especially alongside changing the other test to change the Auto send preference using the Android preference component.